### DISCUSSION TITLE
Add irradiance device class

### DISCUSSION THREAD

**Discussion by epenet** [home-assistant/architecture#836] [Entity Models] [answered]
I propose that we add an `irradiance` sensor device_class to home assistant core and frontend (+ documentation) with supported units:

- `W/m²`
- `BTU/(h.ft²)`

Looking at the use of the unit, these integrations would benefit: `buienradar`, `ecowitt`, `isy994`, `tomorrowio`, `zwave_js`

**Comment by epenet** [comment]
Implementation PR: https://github.com/home-assistant/core/pull/83452

**Comment by emontnemery** [comment, marked as answer]
The proposal is accepted, implementation can be merged when reviewed and approved.
